File: src/main/java/com/sica/backend/dto/AuthRequest.java
package com.sica.backend.dto;

import jakarta.validation.constraints.NotBlank;

public record AuthRequest(
    @NotBlank(message = "El nombre de usuario es requerido")
    String username,

    @NotBlank(message = "La contraseña es requerida")
    String password
) {}

File: src/main/kotlin/com/sica/backend/dto/AuthRequest.kt
package com.sica.backend.dto

import jakarta.validation.constraints.NotBlank

data class AuthRequest(
    @field:NotBlank(message = "El nombre de usuario es requerido")
    val username: String,
    
    @field:NotBlank(message = "La contraseña es requerida")
    val password: String
)
